src/ResourceManager/Maps/Maps.Test/ScenarioTests/MapsAccountTests.ps1

```diff
@@ -49,6 +49,43 @@ function Test-NewAzureRmMapsAccount
     }
 }
 
+<#
+.SYNOPSIS
+Test New-AzureRmMapsAccountS1
+#>
+function Test-NewAzureRmMapsAccountS1
```

Comment:
@SudhindraKovalam this test isn't currently being run as a part of CI -- you'll need to create a new test method that calls this function in the [`MapsAccountTests.cs`](https://github.com/Azure/azure-powershell/blob/master/src/ResourceManager/Maps/Maps.Test/ScenarioTests/MapsAccountTests.cs) file. Once added, you'll need to add the corresponding `.json` file containing the recording for this test that will be used as a part of playback during CI

Reply:
Update: we will enable this test once the work to restructure the repository has been completed.